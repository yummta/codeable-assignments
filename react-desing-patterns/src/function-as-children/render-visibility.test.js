import React from "react";
import { render } from "@testing-library/react";
import RenderVisibility from "./render-visibility";

jest.mock("./get-visibility", function({ cb }) {
  return cb({ isVisible: "Visible", list: [<li>Visible - 00:00:66</li>] });
});

test("RenderVisibility test", () => {
  const { asFragment } = render(<RenderVisibility />);

  expect(asFragment()).toMatchSnapshot();
});
