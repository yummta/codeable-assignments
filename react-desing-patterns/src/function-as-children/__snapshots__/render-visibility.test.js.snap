// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenderVisibility test 1`] = `
<DocumentFragment>
  <h1>
    visible
  </h1>
  <ul>
    <li>
      visible, 2019-6-5 17:28:35
    </li>
  </ul>
</DocumentFragment>
`;
